File: BookStore/BookStore.Api/Models/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using BookStore.Api.Models;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace BookStore.Model
{
    public class Book : ModelBase
    {
      
       
        public string Title { get; set; }
        public string Author { get; set; }
        public bool Check { get; set; }


        public override string ToString()
        {
            return Author + ' '+ Title;
        }

    }
}

File: BookStore/AngularServer/wwwroot/app/scripts/model/book.js
var Book = (function () {
    function Book(id, title, author, check, genre) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.check = check;
        this.genre = genre;
    }
    return Book;
})();
exports.Book = Book;
